keycloak/event-listeners/src/main/java/org/eclipse/sw360/keycloak/event/model/UserEntity.java

```diff
@@ -0,0 +1,184 @@
+/*
+SPDX-FileCopyrightText: © 2024 Siemens AG
+SPDX-License-Identifier: EPL-2.0
+*/
+package org.eclipse.sw360.keycloak.event.model;
+
+import java.util.List;
+import java.util.Map;
+
+public class UserEntity {
+	private String id;
+	private String username;
+	private boolean enabled;
+	private boolean emailVerified;
+	private String firstName;
+	private String lastName;
+	private String email;
+	private List<String> requiredActions;
+	private List<String> groups;
+	private Map<String, List<String>> attributes;
+	private long createdTimestamp;
+	private boolean totp;
+	private List<String> disableableCredentialTypes;
+	private List<String> federatedIdentities;
+	private int notBefore;
+	private Access access;
+	private UserProfileMetadata userProfileMetadata;
+	private String password;
+
+	public String getPassword() {
+		return password;
+	}
+
+	public String setPassword(String password) {
```

Comment:
Can use `@Getter` and `@Setter` from [lombok](https://projectlombok.org/features/GetterSetter) to avoid writing such big classes :wink:

Reply:
Done